common/lib/xmodule/xmodule/graders.py

```diff
@@ -13,18 +19,25 @@
 Score = namedtuple("Score", "earned possible graded section module_id")
 
 
+def float_sum(iterable):
+    """
+    Sum the elements of the iterable, and return the result as a float.
+    """
+    return float(sum(iterable))
```

Comment:
Good question, but no.  The main difference in behavior between floats and ints (in python 2) only happens for division.  All sum does is add them up, so there's no concern about getting incorrect results.